src/test/java/org/galatea/starter/entrypoint/IexRestControllerTest.java

```diff
@@ -125,10 +135,10 @@ public void testGetHistoricalPricesSymbolEmpty() throws Exception {
   public void testGetHistoricalPricesDateAndRangeEmpty() throws Exception {
     MvcResult result = this.mvc.perform(
             org.springframework.test.web.servlet.request.MockMvcRequestBuilders
-                .get("/iex/historicalPrices?token=xyz1&symbol=AAPL")
+                .get("/iex/historicalPrices?token=xyz1&symbol=TSLA")
                 .accept(MediaType.APPLICATION_JSON_VALUE))
         // This URL will be hit by the MockMvc client. The result is configured in the file
-        // src/test/resources/wiremock/mappings/mapping-historicalPrices-range.json
+        // src/test/resources/wiremock/mappings/mapping-historicalPrices-noDateOrRange.json
```

Comment:
Actually, this request will never actually reach your mock, because your own application throws an error before it can.

Reply:
I don't think that is correct - if no date or range parameter is provided, the system defaults to a range of 1 month. This mirrors what IEX does when it is called (would look like this wiremock).